model/src/main/scala/org/broadinstitute/dsde/rawls/model/ErrorReportable.scala

```diff
@@ -12,11 +11,11 @@ trait ErrorReportable {
     throwable match {
       case gjre: GoogleJsonResponseException =>
         val statusCode = StatusCodes.getForKey(gjre.getStatusCode)
-        ErrorReport(ErrorReport.message(gjre), statusCode, ErrorReport.causes(gjre), Seq.empty, Option(gjre.getClass))(errorReportSource)
+        ErrorReport(ErrorReport.message(gjre), statusCode, ErrorReport.causes(gjre)(errorReportSource), Seq.empty, Option(gjre.getClass))(errorReportSource)
```

Comment:
Hmm. That being the case, can you update [this test](https://github.com/broadinstitute/rawls/blob/develop/core/src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala#L546) with whatever's appropriate?